index.html

```diff
@@ -1,55 +1,237 @@
-                <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
-            </ul>
-            <div class="handburger">
-                <span class="bar"></span>
-                <span class="bar"></span>
-                <span class="bar"></span>
-            </div>
-        </nav>
+      <nav class="navbar">
+        <a href="#">
+          <h1 class="nav-title">Welcome</h1>
+        </a>
+        <ul class="nav-menu">
+          <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">About</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
+        </ul>
+        <div class="handburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
     </header>
 
     <section class="first-section">
-        <div class="container">
-            <h1>Hey there. I’m Daniel</h1>
-            <h1 class="desc">I’m a software developer</h1>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
-                <ul class="social">
-                    <li><a href="#"><img src="./Images//twitter.svg" alt=""></a></li>
-                    <li><a href="#"> <img src="./Images//linkedin.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images/medium.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//github.svg" alt=""></a></li>
-                    <li><a href="#"><img src="./Images//angellist.svg" alt=""></a></li>
-                </ul>
-        </div>
+      <div class="container">
+        <h1>Hey there. I’m Daniel</h1>
+        <h1 class="desc">I’m a software developer</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social">
+          <li>
+            <a href="#"><img src="./Images//twitter.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"> <img src="./Images//linkedin.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images/medium.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//github.svg" alt="" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./Images//angellist.svg" alt="" /></a>
+          </li>
+        </ul>
+      </div>
     </section>
-    <!-- <button><img src="./Images//disabled.svg" alt=""></button>
+    <button><img src="./Images//disabled.svg" alt="" /></button>
     <section class="second-section">
+      <div class="container">
```

Comment:
- [ ] Dear @danutibe07, Good job adding the section tag, Although I couldn't find your styling for this specific section (`second-section`). According to the requirement, it's mandatory to use `display: grid` to align this section.
- [ ] This and the `About myself` section both need to be responsive to different screen sizes. I  see You have used a fixed width and that is causing a problem. As you can see your design is not responsive on a screen size `>375px`. Please fix this for a better layout.

![Screenshot 2023-01-13 092546](https://user-images.githubusercontent.com/69077061/212252710-eb40e223-001f-4796-99c0-c0f06a8c1aae.jpg)